samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -514,13 +543,29 @@ static void stream_stopped(struct bt_audio_stream *stream)
 	k_work_cancel_delayable(&audio_send_work);
 }
 
+
+static void stream_enabled_cb(struct bt_audio_stream *stream)
+{
+	/* The unicast server is responsible for starting sink ASEs after the
+	 * client has enabled them.
+	 */
+	if (stream_dir(stream) == BT_AUDIO_DIR_SINK) {
+		const int err = bt_audio_stream_start(stream);
+
+		if (err != 0) {
+			printk("Failed to start stream %p: %d", stream, err);
+		}
+	}
+}
```

Comment:
Why `bt_audio_stream_start` is called for SINK only? The API says:

```
 *  For the unicast server, this will put the stream into the streaming state if
 *  the CIS is connected (initialized by the unicast client). If the CIS is not
 *  connected yet, the stream will go into the streaming state as soon as the
 *  CIS is connected.
```

In other words I would expect `bt_audio_stream_start` will put the stream in streaming state, where both streams SINK and SOURCE need to be handled the same way from API point of view.

It's a detail that in case of SINK and SOURCE this method will be handled differently internally:
SINK:
* Wait for CIS to be connected if not yet
* Notify ASE state: Streaming
* Call stream->started()

SOURCE:
* Wait for CIS to be connected if not yet
* Wait for ASE control operation: Receiver Start Ready
* Notify ASE state: Streaming
* Call stream->started()

Reply:
We could have similar behavior for sink and source ASEs for `bt_audio_stream_start`, but my thought here would be that `bt_audio_stream_start` is a wrapper for the receiver start ready operation, which can only be done on sink ASEs on the server, and similarly only for source ASEs on the client (as per the ASCS spec). 

I agree that the documentation of the function is incorrect in this regard, and should be updated.

Reply:
Updated the documentation a bit :)